packages/firestore/src/api/database.ts

```diff
@@ -283,9 +290,24 @@ export function configureFirestore(firestore: Firestore): void {
     firestore._queue,
     databaseInfo
   );
+  if (
+    settings.cache?._offlineComponentProvider &&
+    settings.cache?._onlineComponentProvider
+  ) {
+    firestore._firestoreClient._uninitializedComponentsProvider = {
+      _offlineKind: settings.cache.kind,
+      _offline: settings.cache._offlineComponentProvider,
+      _online: settings.cache._onlineComponentProvider
+    };
+  }
 }
 
 /**
+ * @deprecated This function will be removed in a future major release. Instead, set
```

Comment:
I'm not sure but I think this tag needs to go at the bottom of the comment block. You can double check when you run yarn docs devsite.

Reply:
Done in the latest commit.